doc/help/man-ippeveprinter.html

```diff
@@ -110,11 +110,13 @@ <h2 class="title"><a name="OPTIONS">Options</a></h2>
 <b>ippeveprinter</b>
 will create an output file using the job ID and name.
 <dt><b>-F </b><i>output-type/subtype[,...]</i>
-<dd style="margin-left: 5.0em">Specifies the output MIME media type.
+<dd style="margin-left: 5.0em">Specifies the output MIME media type in case printer command (parameter "-c") supports more media types.
```

Comment:
How about "Specifies the output MIME media type that is provided to the printer command."?